evaluator/builtin_time_test.go

```diff
@@ -316,6 +316,14 @@ func (s *testEvaluatorSuite) TestCurrentTime(c *C) {
 	c.Assert(err, NotNil)
 }
 
+func (s *testEvaluatorSuite) TestUTCDate(c *C) {
```

Comment:
ok, I will take a look, and maybe modification of current_date() is also needed.